### ISSUE TITLE
UA shop: I purchased today and got an old started date

### ISSUE THREAD

**Issue by anthonydillon** [canonical-web-and-design/ubuntu.com#9648]
![image](https://user-images.githubusercontent.com/1413534/116389736-36acb580-a815-11eb-92d2-0b94b2262e21.png)

**Comment by alnvdl** [comment]
Created on is not the start date of the contract, that is the date on which the contract was created. We have 3 fields:
- `effectiveFrom`: the first date for the current validity period of the contract (which may have been interrupted over time)
- `effectiveTo`: the current expected end date for the contract (assuming no renewals are performed)
- `createdAt`: when the contract was created (this never changes)

The detailed information that could be used to explain what is going on here is present in the `items` field of the contract, which details any purchases that have been made, either in the shop, or directly via sales ops.

**Comment by anthonydillon** [comment]
I agree displaying the items here would really help but also we should consider displaying expired contracts in a separate area that would become active when "renewed". I think that would lead to a much clearer UX.
